### ISSUE TITLE
CompactView botches up

### ISSUE THREAD

**Issue by TheAksh** [TimboKZ/Chonky#47]
A low priority issue, but this basically causes the compact view as unusable.
Additionally, a way to display the date in our own format would be great.

<img width="774" alt="Screen Shot 2020-11-19 at 2 11 44 PM" src="https://user-images.githubusercontent.com/13994976/99712846-98e8d980-2a71-11eb-94d1-bfffe0ede6e9.png">

**Comment by TimboKZ** [comment]
Yeah... The compact view is experimental at the moment, I'm not sure I'm going to keep it as a built-in feature.

The main purpose for the compact view right now is to replace the mobile list view (which is unusable at the moment).
